foundation/foundation-mda/src/main/resources/templates/deployment/argocd/v2/spark.infrastructure.yaml.vm

```diff
@@ -6,6 +6,15 @@ metadata:
   finalizers:
     - resources-finalizer.argocd.argoproj.io
 spec:
+  ignoreDifferences:
+    - group: ""
+      kind: ConfigMap
+      jsonPointers:
+        - /data/metastore-site.xml
```

Comment:
Recommend leaving a comment for why we are doing this (i.e.:  Ignore differences created by the config store injecting values resulting in the resource always being OutOfSync within ArgoCD)

Reply:
added!